OpenMarket/OpenMarket/Controller/MainViewController.swift

```diff
@@ -207,15 +252,45 @@ extension MainViewController {
         collectionView.dataSource = listDataSource
         collectionView.delegate = self
     }
+    
+    private func setupRefreshController() {
+        self.refresher = UIRefreshControl()
+        self.collectionView.alwaysBounceVertical = true
+        self.refresher.tintColor = UIColor.red
+        self.refresher.addTarget(self, action: #selector(loadData), for: .valueChanged)
+        self.collectionView.refreshControl = refresher
+    }
+    
+    
+    
+    private func stopRefresher() {
```

Comment:
```suggestion
    }
    
    private func stopRefresher() {
```
=> 여기도 메소드 간 간격은 한칸으로 해주세요~ :)

Reply:
엇. . 실수가 있었군요 ..! 꼼꼼히 봤어야했는데 ,, ㅜㅜ 수정 했습니다 !!